scripts/install_script.sh

```diff
@@ -0,0 +1,228 @@
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target" | sudo tee "/lib/systemd/system/prometheus.service"
+
+echo "------ Start prometheus -----------"
+
+sudo systemctl daemon-reload
+sudo systemctl enable prometheus.service
+sudo systemctl start prometheus.service
+
+
+echo "-------- Installing node exporter -----------"
+
+cd $HOME
+
+curl -LO https://github.com/prometheus/node_exporter/releases/download/v0.18.1/node_exporter-0.18.1.linux-amd64.tar.gz
+
+tar -xvf node_exporter-0.18.1.linux-amd64.tar.gz
+
+sudo cp node_exporter-0.18.1.linux-amd64/node_exporter $GOBIN
+
+echo "---------- Setup Prometheus Node exporter service -----------"
+
+echo "[Unit]
+Description=Node_exporter
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=$HOME/go/bin/node_exporter
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
+
+[Install]
+WantedBy=multi-user.target" | sudo tee "/lib/systemd/system/node_exporter.service"
+
+echo "----------- Start node exporter ------------"
+
+sudo systemctl daemon-reload
+
+sudo systemctl enable node_exporter.service
+
+sudo systemctl start node_exporter.service
+
+echo "---- Cleaning .dep .tar.gz files of grafana, prometheus, influxdb and node exporter --------"
+
+rm influxdb_1.8.3_amd64.deb grafana_7.3.1_amd64.deb node_exporter-0.18.1.linux-amd64.tar.gz prometheus-2.22.1.linux-amd64.tar.gz
+
+echo "------------Creating databases matic-------------"
+
+curl "http://localhost:8086/query" --data-urlencode "q=CREATE DATABASE matic"
+
+
+echo "--------- Cloning matic-validator-mission-control -----------"
+
+cd $HOME
+
+git clone https://github.com/vitwit/matic-jagar.git
+
+cd matic-jagar
+
+git fetch && git checkout mumbai-testnet
+
+mkdir -p  ~/.matic-jagar/config/
+
+cp example.config.toml ~/.matic-jagar/config/config.toml
+
+cd $HOME
+
+echo "------ Updatig config fields with exported values -------"
+
+sed -i '/eth_rpc_endpoint =/c\eth_rpc_endpoint = "'"$ETH_RPC_ENDPOINT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/bor_rpc_end_point =/c\bor_rpc_end_point = "'"$BOR_RPC_ENDPOINT"'"' ~/.matic-jagar/config/config.toml
+
+sed -i '/bor_external_rpc =/c\bor_external_rpc = "'"$BOR_EXTERNAL_RPC"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/heimdall_rpc_endpoint =/c\heimdall_rpc_endpoint = "'"$HEIMDALL_RPC_ENDPOINT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/heimdall_lcd_endpoint =/c\heimdall_lcd_endpoint = "'"$HEIMDALL_LCD_ENDPOINT"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/heimdall_external_rpc =/c\heimdall_external_rpc = "'"$HEIMDALL_EXTERNAL_RPC"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/validator_hex_addr =/c\validator_hex_addr = "'"$VAL_HEX_ADDRESS"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/signer_address =/c\signer_address = "'"$SIGNER_ADDRESS"'"'  ~/.matic-jagar/config/config.toml
+
+sed -i '/validator_name =/c\validator_name = "'"$VALIDATOR_NAME"'"'  ~/.matic-jagar/config/config.toml
```

Comment:
Is this required? On the dashboard moniker is being displayed by querying the endpoint.

Reply:
Yeah, but in alerts we are displaying moniker from the config.